source/draft_users.rst

```diff
@@ -278,9 +278,16 @@ Delete a user. Returns a 200 OK with empty response body on success, or an
 Role Management
 ---------------
 
-Role management is handled through the ``projects`` endpoints. The projects
-model contains a ``users`` object, which contains three lists: ``members``,
-``spectators``, and ``managers``, each lists of usernames. An admin or project
-manager may set these at any time, adding to or removing from any of the lists.
-A project must always have at least one manager, however. Attempting to remove
-all managers from a project will return an error.
+Role management is handled through the ``projects`` and ``users`` endpoints.
+
+The users model contains the ``site_spectator``, ``site_manager``, and ``site_admin``
```

Comment:
Not to disagree, but we do refer to the objects as "models" in the ``draft_models.rst`` file. It's how we refer to the "class"/"prototype" of an object that can be returned from the API. Maybe it's worth discussing changing, but as it is, this is at least consistent.